alpm/examples/callbacks.rs

```diff
@@ -20,7 +20,7 @@ fn main() {
     // This makes use of the state argument
     // Wrap in an Rc RefCell to allow it to also be read outside the callback
     let number = Rc::new(RefCell::new(0));
-    handle.set_log_cb(number.clone(), |loglevel, msg, data| {
```

Comment:
The goal is to show how you can use Rc/RefCell to share/mutate data passed to callbacks. It's probably best to actually make the clone non redundant by doing something like `println!("file valur of number was: {}`, numer)` at the end of the function.

These tests are also run through valgrind so the Rc code also helps detect leaks.